### PR TITLE
added debug logging for show-libmachine-logs

### PR DESCRIPTION
**Pull request by aaron-prindle**
This helped me debug the hyperv driver, thought I should add it here

### CODE REVIEW

File: cmd/minikube/cmd/root.go [hunk 1]
```diff
@@ -74,6 +79,7 @@ func Execute() {
 
 func init() {
 	RootCmd.PersistentFlags().BoolVarP(&showLibmachineLogs, "show-libmachine-logs", "", false, "Whether or not to show logs from libmachine.")
+	RootCmd.PersistentFlags().BoolVarP(&libmachineDebug, "debug", "", false, "[Enable libmachine debug mode]")
```
Comment (dlorenc) [inline cmd/minikube/cmd/root.go]: What do you think about just defaulting this to true? show-libmachine-logs is already "debug"-y. We probably dont' need two levels.

Comment (aaron-prindle) [inline cmd/minikube/cmd/root.go]: I think that makes sense, I will change it to that.

File: cmd/minikube/cmd/root.go [hunk 2]
```diff
@@ -56,6 +56,7 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
+		log.SetDebug(true)
```
Comment (dlorenc) [inline cmd/minikube/cmd/root.go]: I think this might be slightly more clear (but have the same effect) if you do log.SetDebug(showLibmachineLogs) instead.

**Comment by dlorenc** [comment]
LGTM

**Pull request merged**